CTFd/themes/admin/assets/js/pages/configs.js

```diff
@@ -124,6 +124,44 @@ function updateConfigs(event) {
   });
 }
 
+function switchMode() {
+  let new_mode = $("#user_mode option:selected").val();
+  let body = "";
+  if (new_mode == "teams") {
+    body =
+      "Going from user mode to teams mode will not change anything, but are you sure you want to make the change?";
```

Comment:
This doesn't really convey why it's worth being cautious. I would reword so that the user understands that it's not necessarily the same when switching back.

Reply:
what about: "When switching from user mode to teams mode, user submissions will be kept but users can not solve problems unless on a team.<br> <br>They will also have to solve problems again under the team for the points to go towards the teams' score.<br><br>Are you sure you want to make this change?"